doc/source/tutorial/stats/continuous_jf_skew_t.rst

```diff
@@ -0,0 +1,35 @@
+
+.. _continuous-jf_skew_t:
+
+Jones and Faddy Skew-T Distribution
+===================================
+
+A skew extension of the `t` distribution, defined for :math:`a>0` and
+:math:`b>0`.
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*}
+    f(x;a,b) & = & C_{a,b}^{-1} \left(1+\frac{x}{\left(a+b+x^2\right)^{1/2}}\right)^{a+1/2} \left(1-\frac{x}{\left(a+b+x^2\right)^{1/2}}\right)^{b+1/2} \\
+    F(x;a,b) & = & I\left(\frac{1+x(a+b+x^2)^{-1/2}}{2};a,b\right) \\
+    \mu_{n}^{\prime} & = & \frac{(a+b)^{n/2}}{2^nB(a,b)}\sum_{i=0}^{n}{n \choose i}(-1)^iB\left(a+\frac{n}{2}-i, b-\frac{n}{2}+i\right)
```

Comment:
Seems to match
<img width="546" alt="image" src="https://github.com/scipy/scipy/assets/6570539/0625f090-d161-4661-92fb-9cc5066cbd65">